src/types.ts

```diff
@@ -431,3 +431,13 @@ export type Reload = (
   optHref: DOMString | null | undefined,
   opts: HvComponentOptions,
 ) => void;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type LogFunction = (message?: any, ...optionalParams: any[]) => void;
+
+export type Logger = {
```

Comment:
this might make more sense to be defined within `services/logging`? If we really need it outside of Hyperview, it can be re-exported from this file